File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/readmodel/SeatTypeDTO.cs
using System;

namespace ConferenceManagement.ReadModel
{
    public class SeatTypeDTO
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Quantity { get; set; }
        public decimal Price { get; set; }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/readmodel/SeatTypeDTO.java
package com.microsoft.conference.management.readmodel;

import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
public class SeatTypeDTO {
    private String id;
    private String name;
    private String description;
    private int quantity;
    private BigDecimal price;
}